### DISCUSSION TITLE
Line on sharp edge

### DISCUSSION THREAD

**Discussion by LennieAx** [bnpr/Malt#497] [Q&A] [answered]
Hi I'm new to Malt and it is a tiny bit overwhelming :)

I'm trying to set up a shader with thicker outlines and thinner inlines, so far so good. What I'd also would like to achieve is the ability to have all sharp edges always be drawn with a line, even if they are in the middle of a flat surface..

**Comment by pragma37** [comment, marked as answer]
Hello! Sorry for the super late reply.
Sadly, that's not doable at the moment.

As a workaround, you could assign a different custom ID to each "sharp island" to have lines drawn between them.
See this thread for reference: https://github.com/bnpr/Malt/discussions/296

I have a working prototype that draws lines on edges marked as *freestyle edges* along other geometry based line types, but it's still WIP.

**Reply by LennieAx** [reply]
Hi, no worries, I got sidetracked with another project anyway. 

Do you have any eta on when that may be released?
I did get partially there using multiple materials and setting different id's from them. It got me very close but not all the way.
